packages/shared/src/write-archive/archive-file.ts

```diff
@@ -4,6 +4,12 @@ import { createHash } from 'node:crypto';
 import { logger } from '../utils/logger';
 import type { ArchiveResponse, UrlString } from '../resource-archive';
 
+const RESERVED_PATHNAMES: string[] = [
+  // This is a list of filenames that are reserved for Storybook as a dev target.
+  '/index.json',
+  '/iframe.html',
+];
```

Comment:
What about `index.html`? This is also needed for the manager to work [perhaps less important but still].

Reply:
Also I think this comment could give a bit more detail.